From c0b11808635c87147bb640143c3694f70a935738 Mon Sep 17 00:00:00 2001
From: Michel JAOUEN <michel.jaouen@st.com>
Date: Tue, 7 May 2024 15:03:34 +0200
Subject: [PATCH 0857/1695] plat-stm32mp2: add mailbox driver support

The mailbox driver is enable for the support of
async notifications between CPU2 and OPTEE running
on CPU1.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git bb2b8f5d4b14a79a8af642983b0e6bfb12c99af8 plat-stm32mp2: add mailbox driver support]
Change-Id: Idc53c0cb604a6d3511173f03313b630a0abc635e
Signed-off-by: Michel JAOUEN <michel.jaouen@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/378808
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 583260334..f4736a813 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -85,6 +85,7 @@ $(call force,CFG_CORE_ASYNC_NOTIF_GIC_INTID,31)
 $(call force,CFG_DRIVERS_CLK,y)
 $(call force,CFG_DRIVERS_CLK_DT,y)
 $(call force,CFG_DRIVERS_GPIO,y)
+$(call force,CFG_DRIVERS_MAILBOX,y)
 $(call force,CFG_DRIVERS_PINCTRL,y)
 $(call force,CFG_DRIVERS_REGULATOR,y)
 $(call force,CFG_DT,y)
-- 
2.47.2

